todo_app/data/session_items.py

```diff
@@ -67,3 +67,20 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+
+def delete_item(item):
+    """
+    Deletes an existing item in the session. If no existing item matches the ID of the specified item, nothing is deleted.
+
+    Args:
+        item: The item to delete.
+    """
+    existing_items = get_items()
+
+    for existing_item in range(len(existing_items)):
+      if str(existing_items[existing_item]['id']) == item:
+          del existing_items[existing_item] 
+          break
```

Comment:
Nice solution to this bit! A handy tool for this can be the Python `enumerate` function which lets us loop over both the index number & the item itself at the same time:

```suggestion
    for index, existing_item in enumerate(existing_items):
      if str(existing_item['id']) == item:
          del existing_items[index] 
          break
```